extension/src/test/AlFunctions.test.ts

```diff
@@ -186,6 +232,10 @@ suite("AL Functions Tests", function () {
     test("AL PageExt Xliff", function () {
         let alObj: ALObject.ALObject = new ALObject.ALObject(ALObjectTestLibrary.getPageExt(), true);
         let linesWithTranslation = alObj.codeLines.filter(line => line._xliffIdWithNames);
+        // linesWithTranslation.forEach(line => {
```

Comment:
You want to save this comment block?

Reply:
Yep!
Nice to have when the AL code in the test needs to be updated 🙂

Reply:
just found out that I had a "CodeGenerator" test already. Cleaning up! :)